streams/src/main/java/org/apache/kafka/streams/kstream/internals/StreamSourceNode.java

```diff
@@ -35,25 +35,21 @@
     private final ConsumedInternal<K, V> consumedInternal;
 
 
-    StreamSourceNode(final String parentProcessorNodeName,
-                     final String processorNodeName,
+    StreamSourceNode(final String nodeName,
                      final Collection<String> topics,
                      final ConsumedInternal<K, V> consumedInternal) {
-        super(parentProcessorNodeName,
-              processorNodeName,
+        super(nodeName,
```

Comment:
`topics` -> `topicNames` to be consistent with other `topicName`

Reply:
ack